x-pack/plugins/security_solution/public/cloud_security_posture/components/csp_details/vulnerabilities_findings_details_table.tsx

```diff
@@ -94,19 +97,23 @@ export const VulnerabilitiesFindingsDetailsTable = memo(({ queryName }: { queryN
     }
   };
 
-  const navToVulnerabilities = useNavigateVulnerabilities();
+  const navUrlParams = useGetNavigationUrlParams();
```

Comment:
since `useGetNavigationUrlParams` returns a function its better to name the return value `getNavUrlParams`
so if you want to extract its own return value it would make sense to read as in: `const navUrlParams = get NavUrlParams()`